### PR TITLE
Fix to be able to fully rebuild replicated database

### PR DESCRIPTION
**Pull request by geepie**
- Do not fail on extension objects
- Do not fail on function with no return
- Ignore Row Level Security
- Remove Index Creation on FK
- Provide a script for Index Creation on FK
- Better logging

### CODE REVIEW

File: replayer/connemara_replay.pl [hunk 1]
```diff
@@ -259,65 +259,6 @@ sub get_prep_statement_object
     return $sth;
 }
 
-# This is called when idling. We create missing indexes, in case we got a FK creation for instance
-sub create_missing_indexes
```
Comment (NataliyaLev) [inline replayer/connemara_replay.pl:263]: Can you please give a reason of why you are splitting out reindexing job into a shell script?

**Review by NataliyaLev — COMMENTED** [review]
Can connemara_reindex.sh be used as a stand alone?
Also, please see my comment on connemara_replay.pl.

File: replayer/connemara_replay.pl (hunk 1 above)
Comment (geepie) [reply replayer/connemara_replay.pl:263]: Index on FK is an optimization that requires long lock on tables.
With large queue, locks are longer.
Moving out anything that may lock replay is good.

**Comment by geepie** [comment]
connemara_reindex.sh may be used standalone.
It is feature of connemara, the feature should stay within connemara repo with the same scope.

**Review by NataliyaLev — APPROVED** [review]

**Pull request merged**